crates/connector_configs/src/connector.rs

```diff
@@ -0,0 +1,267 @@
+    pub coinbase: Option<ConnectorTomlConfig>,
+    pub cryptopay: Option<ConnectorTomlConfig>,
+    pub cybersource: Option<ConnectorTomlConfig>,
+    pub iatapay: Option<ConnectorTomlConfig>,
+    pub opennode: Option<ConnectorTomlConfig>,
+    pub bambora: Option<ConnectorTomlConfig>,
+    pub dlocal: Option<ConnectorTomlConfig>,
+    pub fiserv: Option<ConnectorTomlConfig>,
+    pub forte: Option<ConnectorTomlConfig>,
+    pub globalpay: Option<ConnectorTomlConfig>,
+    pub globepay: Option<ConnectorTomlConfig>,
+    pub gocardless: Option<ConnectorTomlConfig>,
+    pub helcim: Option<ConnectorTomlConfig>,
+    pub klarna: Option<ConnectorTomlConfig>,
+    pub mollie: Option<ConnectorTomlConfig>,
+    pub multisafepay: Option<ConnectorTomlConfig>,
+    pub nexinets: Option<ConnectorTomlConfig>,
+    pub nmi: Option<ConnectorTomlConfig>,
+    pub noon: Option<ConnectorTomlConfig>,
+    pub nuvei: Option<ConnectorTomlConfig>,
+    pub payme: Option<ConnectorTomlConfig>,
+    pub paypal: Option<ConnectorTomlConfig>,
+    pub payu: Option<ConnectorTomlConfig>,
+    pub placetopay: Option<ConnectorTomlConfig>,
+    pub plaid: Option<ConnectorTomlConfig>,
+    pub powertranz: Option<ConnectorTomlConfig>,
+    pub prophetpay: Option<ConnectorTomlConfig>,
+    pub riskified: Option<ConnectorTomlConfig>,
+    pub rapyd: Option<ConnectorTomlConfig>,
+    pub shift4: Option<ConnectorTomlConfig>,
+    pub stripe: Option<ConnectorTomlConfig>,
+    pub signifyd: Option<ConnectorTomlConfig>,
+    pub trustpay: Option<ConnectorTomlConfig>,
+    pub tsys: Option<ConnectorTomlConfig>,
+    pub volt: Option<ConnectorTomlConfig>,
+    #[cfg(feature = "payouts")]
+    pub wise_payout: Option<ConnectorTomlConfig>,
+    pub worldline: Option<ConnectorTomlConfig>,
+    pub worldpay: Option<ConnectorTomlConfig>,
+    pub zen: Option<ConnectorTomlConfig>,
+    pub square: Option<ConnectorTomlConfig>,
+    pub stax: Option<ConnectorTomlConfig>,
+    pub dummy_connector: Option<ConnectorTomlConfig>,
+    pub stripe_test: Option<ConnectorTomlConfig>,
+    pub paypal_test: Option<ConnectorTomlConfig>,
+}
+
+impl ConnectorConfig {
+    fn new() -> Result<Self, String> {
+        #[cfg(all(
+            feature = "production",
+            not(any(feature = "sandbox", feature = "development"))
+        ))]
+        let config = toml::from_str::<Self>(include_str!("../toml/production.toml"));
+        #[cfg(all(
+            feature = "sandbox",
+            not(any(feature = "production", feature = "development"))
+        ))]
+        let config = toml::from_str::<Self>(include_str!("../toml/sandbox.toml"));
+        #[cfg(feature = "development")]
+        let config = toml::from_str::<Self>(include_str!("../toml/development.toml"));
+
+        #[cfg(not(any(feature = "sandbox", feature = "development", feature = "production")))]
+        return Err(String::from(
+            "Atleast one features has to be enabled for connectorconfig",
+        ));
```

Comment:
Can we use the [`compile_error!()`](https://doc.rust-lang.org/std/macro.compile_error.html) macro instead to raise an error at compile-time error?